Classes/Notifications/NotificationCell.swift

```diff
@@ -98,6 +100,10 @@ final class NotificationCell: SwipeSelectableCell {
         textLabel.attributedText = viewModel.title.attributedText
         dateLabel.setText(date: viewModel.date)
         reasonImageView.image = viewModel.type.icon?.withRenderingMode(.alwaysTemplate)
+        accessibilityLabel = contentView.subviews
+            .flatMap { $0.accessibilityLabel }
+            .reduce("", { $0 + ".\n" + $1 })
```

Comment:
the `./n`-s makes VoiceOver read these as sentences instead of just one, long sentence, making the different labels / elements separated.